### PR TITLE
remove duplicate

### PR DESCRIPTION
**Pull request by msusman1**
remove test comments

### CODE REVIEW

**Comment by msusman1** [comment]
why you duplicate code, we need that

**Comment by arman229** [comment]
that was for testing

File: src/bmiComponents/BmiApp.js [hunk 1]
```diff
@@ -16,12 +17,8 @@ export default function BmiApp() {
                     Calculator</Typography>
                 <CardContent>
                     <BmiInput/>
-                    <VerticalSpace/>
-                    <BmiInput/>
                 </CardContent>
             </Card>
-
-<p>dd</p>
```
Comment (arman229) [inline src/bmiComponents/BmiApp.js:24]: this should not be removed this is for testing

**Comment by msusman1** [comment]
okay

**Pull request merged**